flask/user.py

```diff
@@ -533,7 +533,139 @@ def remove_maintainers_from_namespace(username):
         return jsonify({"message": "Maintainer removed successfully", "code": 200}), 200
     else:
         return jsonify({"message": "Namespace maintainer not found", "code": 200}), 200
+    
+@app.route("/<username>/namespace/admin", methods=["POST"])
+def add_admins_to_namespace(username):
+    uuid = request.form.get("uuid")
+    username_to_be_added = request.form.get("username")
+    namespace = request.form.get("namespace")
```

Comment:
I'm not an expert on this, but what if I was able to mock the request and send another valid namespace name as form data, which I'm not an admin of. Would I be able to add admins to any namespace at will?

Reply:
No, because this check will stop the user from adding admins 
https://github.com/fortran-lang/registry/pull/29/files#diff-dda7547301513b9e04c8c3392ca97c1855def429c543bd410c86b80335f22672R573

Reply:
Ok 👍🏽